js/client.js

```diff
@@ -22,20 +22,23 @@ class Client {
 
   // When data is received from the server
   receive(messageData) {
```

Comment:
hmm, why are you changing this method so much? my guess is you accidentally reverted while merging